Library/Homebrew/cmd/install.rb

```diff
@@ -261,7 +261,9 @@ def install_formula(f)
     f.print_tap_action
 
     fi = FormulaInstaller.new(f)
-    fi.options             = f.build.used_options
+    build_options          = f.build
```

Comment:
Also, wondering if we'll want anything like this in `reinstall` or `upgrade`? I figure probably not but worth thinking about.